### ISSUE TITLE
Document wait() in API.md

### ISSUE THREAD

**Issue by JHKennedy4** [emberjs/ember-test-helpers#306]
![screen shot 2018-01-23 at 11 20 22 am](https://user-images.githubusercontent.com/159719/35295929-6d8c2f7c-002f-11e8-9d13-bbe05aff761e.png)

https://embercommunity.slack.com/archives/C045C9ABM/p1516735003000397

**Comment by shankarsridhar** [comment]
I see `settled()`, but I can't seem to find `wait()` inside 
https://github.com/emberjs/ember-test-helpers/tree/master/addon-test-support/%40ember/test-helpers 
or 
https://github.com/emberjs/ember-test-helpers/tree/master/addon-test-support/%40ember/test-helpers/dom

Is `wait()` still available?

**Comment by rwjblue** [comment]
`wait` is not (and has never been) exposed from `@ember/test-helpers` module. It was only available from the (now defunct/deprecated) `ember-test-helpers` module.

**Comment by shankarsridhar** [comment]
Thanks for the clarification. In that case, can this issue be closed ?

**Comment by jgomo3** [comment]
@rwjblue What should [this code](https://github.com/elwayman02/ember-user-activity/blob/017fc47c07443c1eb36ff9d891293eef86d4879a/tests/unit/services/user-idle-test.js#L50) use instead of `wait`?

Adding on top of that, a deprecation warning when using the [ember-test-helpers.wait function](https://github.com/elwayman02/ember-user-activity/blob/017fc47c07443c1eb36ff9d891293eef86d4879a/tests/unit/services/user-idle-test.js#L50) could be useful. Even more: one explaining how to replace the wait function in the warned code.

**Comment by rwjblue** [comment]
> @rwjblue What should this code use instead of wait?

```js
import { settled } from '@ember/test-helpers';

// ...snip...
  return settled().then(function () {
    assert.ok(service.get('isIdle'), 'isIdle is set to true after timeout');
  });
```

> Adding on top of that, a deprecation warning when using the ember-test-helpers.wait function could be useful

Yep, agree! We should totally deprecate `wait` usage.

**Comment by SergeAstapov** [comment]
Looks like this can be closed as wait was removed in #818

**Comment by rwjblue** [comment]
Thank you @SergeAstapov
